HW_4/task_3.py

```diff
@@ -0,0 +1,28 @@
+"""
+Задание №8
+? Создайте несколько переменных заканчивающихся и не оканчивающихся на «s».
+? Напишите функцию, которая при запуске заменяет содержимое переменных
+оканчивающихся на s (кроме переменной из одной буквы s) на None.
+? Значения не удаляются, а помещаются в одноимённые переменные без s на конце.
+"""
+
+
+def func():
+    temp_dict = {}
+    my_dict = {key: value for key, value in globals().items() if key.endswith('s') and key != 's'}
+    for key, value in my_dict.items():
+        temp_dict[str(key)[:-1]] = value
```

Comment:
Ключи уже имеет строковый тип
```suggestion
        temp_dict[key[:-1]] = value
```